JapaneseFilter/sections/specific.txt

```diff
@@ -1069,6 +1069,7 @@ hrs-game.main.jp,main-hrs-game.ssl-lolipop.jp##.ads_970x90
 valorantnews.jp##.header-cls
 valorantnews.jp###amgatti_sk-2
 monayoblog.com##.kaisetsu-box2
+appmedia.jp##div[id^="ad_jack"]
```

Comment:
Please move it to AdGuard Annoyances - self-promo.

Reply:
done